tests/app/db/test_db_init.py

```diff
@@ -35,21 +35,31 @@ async def test_init_db(mock_create_async_engine: Mock, read_uri_value: str) -> N
 
     await init_db()
 
-    assert mock_create_async_engine.call_count == 2 if read_uri_value else 1
-    # assert mock_conn.run_sync.call_count == 2 if DB_READ_URI else 1
-
-
-@patch('app.db.db_init._engine_write', spec=AsyncMock)
-@patch('app.db.db_init._engine_read', spec=AsyncMock)
-async def test_close_db(mock_engine_read: AsyncMock, mock_engine_write: AsyncMock) -> None:
+    assert mock_create_async_engine.call_count == 4 if read_uri_value else 2
+
+
+@patch('app.db.db_init._engine_enp_write', spec=AsyncMock)
+@patch('app.db.db_init._engine_enp_read', spec=AsyncMock)
+@patch('app.db.db_init._engine_napi_write', spec=AsyncMock)
+@patch('app.db.db_init._engine_napi_read', spec=AsyncMock)
+async def test_close_db(
```

Comment:
The value of this test isn't obvious to me.  It's an implementation test, and almost everything is mocked.  Is there a reason not to open some ENP db connections, call close_db, and then test that the connection is actually closed?